rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -551,26 +579,60 @@ def _check_input_dimension_consistency(self, model_data: RasaModelData) -> None:
                 )
 
     def _extract_labels_precomputed_features(
```

Comment:
It didn't have tests before - and I think before investing time to write tests it might be better to collect all of these functions in one place (model data utils has very similar sounding functions already and message has a lot of filter and combine functions) and probably deduplicate them a bit?